ctgan/synthesizers/ctgan.py

```diff
@@ -238,7 +238,7 @@ def _cond_loss(self, data, c, m):
                     st = ed
                     st_c = ed_c
 
-        loss = torch.stack(loss, dim=1)
+        loss = torch.stack(loss, dim=1)  # noqa: PD013
```

Comment:
This line gives this error: `PD013 '.melt' is preferred to '.stack'; provides same functionality`.

But torch has no melt function, so I'm guessing the addon just goes through the code looking for the .stack keyword and assumes it's from pandas, so I'm ignoring it.

Reply:
Yes I had the same issue previously, it's okay to noqa this